containers/Staking/LockForm/CreateLock/index.js

```diff
@@ -185,6 +185,7 @@ const CreateLock = () => {
           <ContainedButton
             fullWidth
             type='submit'
+            onClick={() => logSubmit()}
```

Comment:
onClick={logSubmit}

Reply:
^ agreed - if that's changed it looks good to me